surface/include/pcl/surface/marching_cubes.h

```diff
@@ -447,6 +456,15 @@ namespace pcl
       /** \brief The iso level to be extracted. */
       float iso_level_;
 
+      /** \brief ignore the distance function
+       * if it is negative
+       * or distance between voxel centroid and point are larger that it. */
+      float dist_ignore_;
+
+      /** \brief the point cloud really generated from Marching Cubes
+       */
+      pcl::PointCloud<PointNT> intermediate_cloud_;
```

Comment:
With the changes requested I think this member can be removed and just exist in a temporary scope inside the methods.